src/main/java/com/denizenscript/denizencore/utilities/ReflectionHelper.java

```diff
@@ -66,13 +67,20 @@ public static <T> T getFieldValue(Class clazz, String fieldName, Object object)
     public static class CheckingFieldMap extends HashMap<String, Field> {

```

Comment:
this should probably stop extending HashMap if it's not actually using the underlying hashmap anymore